hologram_dashboard/extension/src/app/time/time.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-time',
+  templateUrl: './time.component.html',
+  styleUrls: ['./time.component.css']
+})
+export class TimeComponent implements OnInit {
+  time: String;
+
+  constructor() { }
+
+  ngOnInit(): void {
+    this.time = "TODO";
```

Comment:
nit: or just use a hard-coded time string?

Reply:
@sarahchen-google told me to do this instead of a hard-coded time string.